### PR TITLE
Adds rival corp voidsuits

### PR DESCRIPTION
**Pull request by kyres1**
This PR adds a set of non-NT voidsuits to the uplink for rather high telecrystal prices, varying by suit.

The idea here is to further enable mercenary/traitor gimmicks pertaining to the corporations other than NanoTrasen and spew some lore while we're at it. As well, it will enable the antagonists possessing them to adapt to situations using different armor just as Security can. For example, rather than a singular cruddy heavy vest for 4 TC, you can blow around 3-4x that and get something tailored to your needs versus lasers, melee or otherwise.

All suits are only wearable by Humans with exceptions for Humanoid IPC chassis (like shells, bishops and zeng-hus.)

The suits are as follows :
**Hephaestus** : Caiman drop suit. Big melee resist, slows you lots.
**Einstein** : Banshee combat suit. Big laser resist, low every other resist.
**Zeng-hu** : Dragon biohazard control suit. 100% rad protection and some okay melee resist.
**Necropolis** : Revenant combat suit. A little better than officer vest level bullet protection, low resists elsewhere. Backpack portable.

To-do:
- [x] Changelog

- [x] Adding the suits to the warehouse/merchant spawn pool.

corp bloat

### CODE REVIEW

**Comment by kyres1** [comment]
!wip

**Comment by kyres1** [comment]
!review

**Comment by kyres1** [comment]
> 
> 
> can this be done with contained sprites instead? it would solve .dmi bloat. also, make a new uplink for the suits instead of just hurling all of them into tools

sure

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 1]
```diff
@@ -3,6 +3,38 @@
 	name = "gear crate"
 	req_access = list(access_syndicate)
 
+/obj/structure/closet/crate/gear_loadout/hephaestus
+	desc = "A sturdy crate with Hephaestus Industries branding."
+	name = "hephaestus drop crate"
+	icon_state = "hephcrate"
+	icon_opened = "hephcrateopen"
+	icon_closed = "hephcrate"
+	layer = OBJ_LAYER - 0.01
```
Comment (Arrow768) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm]: Is there a specific reason you added this all over the place?
And if so, why is it not set at one of the parent types?

File: code/game/objects/random/random.dm [hunk 2]
```diff
@@ -442,7 +446,11 @@
 		/obj/item/clothing/suit/space/void/hos = 0.3,
 		/obj/item/clothing/suit/space/void/lancer = 0.3,
 		/obj/item/clothing/suit/space/void/sci = 2,
-		/obj/item/clothing/suit/space/void/sol = 0.5
+		/obj/item/clothing/suit/space/void/sol = 0.5,
+		/obj/item/clothing/suit/space/void/necropolis = 1,
+		/obj/item/clothing/suit/space/void/einstein = 1,
+		/obj/item/clothing/suit/space/void/hephaestus = 1,
+		/obj/item/clothing/suit/space/void/zenghu = 1
```
Comment (Arrow768) [inline code/game/objects/random/random.dm]: These probabilities seem relatively high compared to the other items in the list.

**Pull request merged**